### DISCUSSION TITLE
Separate the package in two parts to get less dependencies

### DISCUSSION THREAD

**Discussion by dvd-bnc** [singerdmx/flutter-quill#377] [Feature Request]
As of now flutter_quill has a lot of dependencies for the toolbar, i was wondering if the toolbar impl can just be moved to another optional package so if you want to implement your custom toolbar you don't have to depend on that many packges.

I'm eventually considering using the package for one of my apps but as of now the dependencies are the issue, i'd end up shipping more than i'd actually use

**Comment by singerdmx** [comment]
Submit a PR for that 
Thanks

**Comment by singerdmx** [comment]
We will have only one package to ship
Two package is too much to manage
